apps/molgenis-components/src/components/tables/RenderValue.vue

```diff
@@ -7,7 +7,7 @@
       />
       <EmailDisplay v-else-if="col.columnType === 'EMAIL_ARRAY'" :data="val" />
       <span v-else>
-        {{ col.refLabel ? applyJsTemplate(col.refLabel, val) : val }}
+        {{ getLabel(col) ? applyJsTemplate(getLabel(col), val) : val }}
```

Comment:
I think this ternary can also go into a function, in order to have less logic in the html